src/components/__tests__/social.test.js

```diff
@@ -0,0 +1,13 @@
+import React from "react"
+import renderer from "react-test-renderer"
```

Comment:
Thanks for your efforts, looks good!
But we shall rely only on one testing library use `enzyme-to-json` library and do not use `react-test-renderer`  - there is no trace of it in `master` now. How to work with it you can see [here](https://github.com/rahuldkjain/github-profile-readme-generator/blob/master/src/components/__tests__/work.test.js#L22)

Reply:
Thank you, I will continue to work on it

Reply:
I have written the test and it now passes fine
![image](https://user-images.githubusercontent.com/45674928/97122411-ebaacc00-1725-11eb-96a8-9139fa405887.png)